src/installer.ts

```diff
@@ -97,6 +113,20 @@ export async function getGo(
   return downloadPath;
 }
 
+async function resolveVersionFromManifest(
+  versionSpec: string,
+  stable: boolean,
+  auth: string | undefined
+): Promise<string | undefined> {
+  try {
+    const info = await getInfoFromManifest(versionSpec, stable, auth);
+    return info?.resolvedVersion;
+  } catch (err) {
+    core.info('Unable to resolve version from manifest...');
```

Comment:
```suggestion
    core.info('Unable to resolve a version from the manifest...');
```

Reply:
Looks like the tests will need to be updated as well since they try to match this string.

Reply:
Done. Thank you.